tools/lambda-promtail/lambda-promtail/s3_test.go

```diff
@@ -652,64 +682,71 @@ func TestProcessSQSEvent(t *testing.T) {
 	require.True(t, handlerCalled)
 }
 
-func TestToMicroseconds(t *testing.T) {
+func TestGetUnixSecNsec(t *testing.T) {
```

Comment:
this test needs a case for the `fractionalSec == 0` case as well

Reply:
[This](https://github.com/grafana/loki/pull/10416/files#diff-f89d8b6a0bfefcada6292d108d96a954c426fa6850340440149542bb3ab613b4R696-R704) tests `fractionalSec == 0`.